src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
@janvorli Yeah, unfortunately the attribute declaration doesn't seem to survive across the archival step (or at least its not respected).  That said, I tend to agree with you -- if there is enough dead code in the .a that it makes a tangible difference on the output of the shared object, we should remove it ipso facto, rather than relying on the linker to do it for us.